<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorOutputWidget</class>
 <widget class="QWidget" name="InspectorOutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutChannel">
     <item>
      <spacer name="horizontalSpacerChannel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelChannel">
       <property name="text">
        <string>&lt;a href=&quot;Channel&quot; style=&quot;color: white;&quot;&gt;Channel&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxChannel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutVideolayer">
     <item>
      <spacer name="horizontalSpacerVideolayer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelVideolayer">
       <property name="text">
        <string>&lt;a href=&quot;Videolayer&quot; style=&quot;color: white;&quot;&gt;Videolayer&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxVideolayer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDelay">
     <item>
      <spacer name="horizontalSpacerDelay">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDelay">
       <property name="text">
        <string>&lt;a href=&quot;Delay&quot; style=&quot;color: white;&quot;&gt;Delay&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDelay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAllowGpi">
     <item>
      <spacer name="horizontalSpacerAllowGpi1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAllowGpi">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;Allow Gpi&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#ffffff;&quot;&gt;Allow GPI&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAllowGpi">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerAllowGpi2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerOutput">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxChannel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>channelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxVideolayer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>videolayerChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDelay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>delayChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDelay</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>resetDelay(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelVideolayer</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>resetVideolayer(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelChannel</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>resetChannel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAllowGpi</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>allowGpiChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelAllowGpi</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>resetAllowGpi(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>channelChanged(int)</slot>
  <slot>videolayerChanged(int)</slot>
  <slot>delayChanged(int)</slot>
  <slot>resetDelay(QString)</slot>
  <slot>resetVideolayer(QString)</slot>
  <slot>resetChannel(QString)</slot>
  <slot>allowGpiChanged(int)</slot>
  <slot>resetAllowGpi(QString)</slot>
 </slots>
</ui>
